packages/ui/src/components/Command/CommandMenu.tsx

```diff
@@ -67,6 +67,12 @@ const CommandMenu = ({ projectRef }: CommandMenuProps) => {
     useCommandMenu()
   const showCommandInput = !currentPage || !CHAT_ROUTES.includes(currentPage)
 
+  // This function has been added to prevent the use of double quotes in the search docs input due to an issue with the cmdk-supabase module. This function can be removed when we transition to using cmdk.
+  const handleInputChange = (value: string) => {
+    const newValue = value.replace(/"/g, '') // Remove double quotes
+    setSearch(newValue)
+  }
+
```

Comment:
Thanks @nwaliaez! Can you help me understand what specifically triggers the error? Is it something in `CommandDialog` that crashes when `value` contains a double quote character?

Reply:
Hello @gregnr, based on my interpretation, the problem lies within the `CommandInput`. It appears to be causing issues with the `data-value` attribute when a double quote is entered. This leads to a `SyntaxError: The execution of 'querySelector' on 'Element' has failed because '[cmdk-item=""][data-value="search the docs: ""]' is an invalid selector.`

Reply:
Got it thank you. I think this is a good interim solution until it's fixed in the lib. Thanks!

Reply:
Thank you @gregnr